apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -365,6 +366,7 @@ def evaluate_with_source(
     # for exponentiation. In Python, only the latter can be used.
     # Also replace the unicode minus with the normal one.
     expr = expr.replace("^", "**").replace("\u2212", "-")
+    expr = unidecode(expr)
```

Comment:
I was reading [the documentation](https://github.com/avian2/unidecode#module-content), and I'd argue that it makes more sense to use:
```suggestion
    expr = unidecode(expr, errors='preserve')
```

In essence, if a character that the user typed cannot be converted, it would make more sense to keep the character rather than ignore it. That said, I don't mind if others believe otherwise. Thoughts?

Reply:
That makes sense, the character will cause the parser throw an error so having it ignore certain characters might cause unexpected behavior.

Reply:
Apologies for the confusion, but it seems the package we're using is `text-unidecode`, not `unidecode`. If I recall correctly, this was done due to license concerns. `text-unidecode` does not support the `errors` option.